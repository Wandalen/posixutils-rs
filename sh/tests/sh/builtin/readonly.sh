readonly a=value
echo $?
echo $a
readonly -p

readonly b=val2 c=val2
echo $?
echo $b $c
readonly -p

var=value
echo $var
readonly var
echo $?
echo $var
readonly -p

echo $var2
readonly var2
echo $var2
readonly -p