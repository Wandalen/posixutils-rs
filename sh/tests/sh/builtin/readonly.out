0
value
readonly a='value'
0
val2 val2
readonly a='value'
readonly b='val2'
readonly c='val2'
0
value3
readonly a='value'
readonly b='val2'
readonly c='val2'
readonly d='value3'
value
0
value
readonly a='value'
readonly b='val2'
readonly c='val2'
readonly d='value3'
readonly var='value'


readonly a='value'
readonly b='val2'
readonly c='val2'
readonly d='value3'
readonly var='value'
readonly var2
